src/pages/NotFound.spec.js

```diff
@@ -1,8 +1,18 @@
 import React from "react";
-import NotFound from "./NotFound";
+import { createMuiTheme } from "@material-ui/core";
+
+import ErrorComponent from "../components/Errors/ErrorComponent";
+import { renderWithTheme } from "../test-utils";
+
+const theme = createMuiTheme({});
 
 describe("NotFound page", () => {
     it("should render successfully", () => {
-        mount(<NotFound />);
+        const errorMessage = "Sorry, no page could be found at this address (404)";
+        const wrapper = renderWithTheme(
+            <ErrorComponent Message={errorMessage}/>,
```

Comment:
In order to test the Not Found page, you should not render its contents, but the page itself, and assert that the contents are as expected. Change the component to just the <NotFound> and assert that the error message is whatever you'll put on the component above. If you add the title, add here the assertion that the title is present as well.

Finally, add a similar test for the error page as well